src/utils/moderation.rs

```diff
@@ -399,3 +400,8 @@ pub async fn mute_moderation(
 
     Ok((is_currently_muted, removed_roles))
 }
+
```

Comment:
I'd consider this a general utility function, not just moderation.

Reply:
which file is appropiate for that? I see that the all the files in the utils dir are for specific scopes

Reply:
I think mod.rs inside utils shoud suffice